index.html

```diff
@@ -44,6 +44,155 @@ <h2 class="name1">I’am a Software Developer</h2>
       </ul>
     </div>
   </section>
+
+  <section>
+    <div class="workcard">
+        <h1 class="recent">My Recent Works</h1>
+
+        <div class="barQf"><div id="barQf"></div></div>
+        <div><img src="img/Img Placeholder.png" alt="" class="imgHOLD"></div>
+
+     <div class="storiesQ">
+      <h2 class="stories"> Multi-Post Stories</h2>
+      <p class="paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+      
+    <div class="tags">
+      <ul>
+        <li class="tag css">css</li>
+        <li class="tag html">html</li>
+        <li class="tag bootstrap">bootstrap</li>
+        <li class="tag ruby">Ruby</li>
+      </ul>
+      </div>
+      <button class="projects"><span class="see-projects">See project</span></button>
+    </div>
+    </div>
+
+     <div class="showcase">
+       <div class="Art">
+           <h2 class="stories1"> Profesional Art Printing Data</h2>
+           <p class="paragraph1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+         <div class="tags1">
+          <ul>
+            <li class="tag html1">html</li>
+            <li class="tag bootstrap1">bootstrap</li>
+            <li class="tag ruby1">Ruby</li>
+          </ul>
+         </div>
+           <button class="projects1"><span class="see-projects1">See project</span></button>
```

Comment:
- You did a great job by adding hover in your button however the hover design is not the same  Kindy  I suggest you use the appropriate color and effect for your button as it was required by the previous reviewer [here](https://github.com/Ellyboi/Mobile-portfolio-website/pull/3/files#r1038026418) are the various state (hover, active and disabled) 
![](https://user-images.githubusercontent.com/73607512/205277421-5a5560cb-3bdc-4966-8f59-cfcec0915e92.PNG)